.github/workflows/frontend.yml

```diff
@@ -42,3 +47,23 @@ jobs:
           yarn test
         env:
           CI: true
+
+  deploy-storybook:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
So I think we still want to build the storybook in PRs as well, but only publish from master, although we can add PR builds later. Either way, I'd move this entire section into a new workflow file and leave this one as-is. The problem with having it here is that we now run the above job on master builds too, which we don't really want, we have a separate workflow for that.

Reply:
okey got it @Rugvip 👍🏻